compiler/rustc_infer/src/infer/lexical_region_resolve/mod.rs

```diff
@@ -707,10 +707,10 @@ impl<'cx, 'tcx> LexicalResolver<'cx, 'tcx> {
                         .constraints
                         .iter()
                         .filter_map(|(constraint, origin)| match (constraint, origin) {
-                            (
-                                Constraint::VarSubVar(_, sup),
-                                SubregionOrigin::DataBorrowed(_, sp),
-                            ) if sup == &node_vid => Some(*sp),
```

Comment:
Could this be removed rather than commented?

Reply:
Done. I've removed the whole function (this was the only thing in it). I believe that this file is from the time of lexical lifetimes and is only kept around for some data structures or something.